test/js/bun/shell/throw.test.ts

```diff
@@ -0,0 +1,50 @@
+import { $ } from "bun";
+import { beforeAll, describe, test, expect } from "bun:test";
+
+beforeAll(() => {
+  $.nothrow();
```

Comment:
Is the idea that `$.nothrow()` and `$.throws(false)` does the same?

Reply:
yep

Reply:
Having two ways of doing the same thing will create confusion down the line. 

My input: only be able to set the global behavour using `.throws(Boolean)` while using `.noThrow()` for individual calls. 

(this is of cause not a "hard" demand, but a request to reconsider)

Reply:
There's inspiration to look at in test runners I wanna say for similar kinds of global vs test level APIs.